### DISCUSSION TITLE
pdfjs package on ArchLinux

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#6076] [Q&A] [answered]
When I did `:set content.pdfjs true` and tried opening a PDF with qutebrowser, I got the _No pdf.js installation found_ message.

I tried searching for `pdfjs` in the AUR, but the only result I could find is [chromium-pdfjs-git](https://aur.archlinux.org/packages/chromium-pdfjs-git/), and after installing it and restarting qutebrowser I get the same error message.

Since that message says

> If you have installed a pdf.js package and qutebrowser still can't find it, please send us a report with your system and the package name, so we can add it to the list of supported packages.

here I am asking for support.

**Comment by bitraid** [comment, marked as answer]
Check official packages first: https://archlinux.org/packages/community/any/pdfjs/
